test.rb

```diff
@@ -0,0 +1,45 @@
+require_relative('lens_memprof')
+include LensCMemprof
+puts allocations
+puts "=" * 100
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.